src/SDKs/Dns/Management.Dns/Microsoft.Azure.Management.Dns.csproj

```diff
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <PackageId>Microsoft.Azure.Management.Dns</PackageId>
     <Description>Provides management capabilities for Azure Dns.</Description>
-    <VersionPrefix>1.9.0-preview</VersionPrefix>
+    <VersionPrefix>1.10.0-preview</VersionPrefix>
```

Comment:
@karravi-msft missing ReleaseNotes. Please model after 
https://github.com/Azure/azure-sdk-for-net/blob/psSdkJson6/src/SDKs/BatchAI/Management.BatchAI/Microsoft.Azure.Management.BatchAI.csproj#L10

Reply:
resolved.. thanks for the pointer